deployability/modules/provision/models.py

```diff
@@ -11,7 +11,7 @@ class ComponentInfo(BaseModel):
     type: str = "package"
     version: str = ""
     dependencies: dict | None = None
-    live : bool = False
+    live : bool = True
```

Comment:
Default packages should be development

Reply:
Done in [a7aa606](https://github.com/wazuh/wazuh-qa/pull/5392/commits/a7aa6062b7d70d81d64975f698f29e7d4959c353)